lib/pages/happiness_exercise.rb

```diff
@@ -38,16 +38,11 @@ def open_review
   end
 
   def has_previous_responses?
-    find('.row', text: "(#{(Date.today - 4).strftime('%m/%d/%Y')}) Please " \
-                       'describe three good things that happened to you today:')
+    find('.row', text: "(#{(Date.today - 4).strftime('%m/%d/%Y')})")
```

Comment:
@a-ashbeck these should be: `find('.row', text: (Date.today - 4).strftime('%m/%d/%Y')` since there is no other text we don't need to interpolate, `strftime` returns a string already